torch/optim/sgd.py

```diff
@@ -498,9 +511,11 @@ def _fused_sgd(
         torch._fused_sgd_(
             device_params,
             device_grads,
-            []
-            if no_momentum_buffer
-            else cast(List[Tensor], device_momentum_buffer_list),
+            (
```

Comment:
what's the linter that you have? These PRs would be easier to review if the lint changes could be excluded.